.github/scripts/actionlint.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+#################################################################
+#    Lint workflows with https://github.com/rhysd/actionlint    #
+#################################################################
+
+# v1.7.7
+readonly ACTIONLINT_VERSION=03d0035246f3e81f36aed592ffb4bebf33a03106
+
+# Verify that shellcheck is installed (preinstalled on GitHub Actions runners)
+if [[ -z "$(command -v shellcheck)" ]]; then
```

Comment:
This can be simplified

```suggestion
if ! command -v shellcheck &>/dev/null; then
```